linera-base/src/ownership.rs

```diff
@@ -198,13 +198,13 @@ pub enum ChangeApplicationPermissionsError {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::crypto::AccountSecretKey;
+    use crate::crypto::{Ed25519SecretKey, Secp256k1SecretKey};
 
     #[test]
     fn test_ownership_round_timeouts() {
-        let super_pub_key = AccountSecretKey::generate().public();
+        let super_pub_key = Ed25519SecretKey::generate().public();
         let super_owner = Owner::from(super_pub_key);
```

Comment:
Don't we want the `Owner` type to be derived from the general `PublicKey` enum? Or is this step temporary, until `Owner` gets replaced by a new address type?

Reply:
But if `Owner` can only be derived from an Ed25519 key, then how can a secp256k1 key ever sign a block proposal?

Reply:
I think Owner can be derived from both. There are impls for `Secp256k1` types as well.